.circleci/config.yml

```diff
@@ -1,16 +0,0 @@
-version: 2
-jobs:
-  build:
-    docker:
-      - image: sphinxdoc/docker-ci
```

Comment:
This image is a key point. The image on https://hub.docker.com/r/sphinxdoc/docker-ci contains LaTeX binaries.

Reply:
Ahh. I'll close this PR then.

A